### PR TITLE
[FIX] hr_skills: fix resume end date wrong selection

### PR DESCRIPTION
**Pull request by pash-odoo**
before this commit, in resume we are able to put end date less than start date
which is not possible in real life.
make change in constraint to check selected end date is greater than start date.
make changes in demo data of end date which are according to new changes.

task-3397726

### CODE REVIEW

File: addons/hr_skills/data/hr.resume.line.csv [hunk 1]
```diff
@@ -3,94 +3,94 @@ employee_resume_admin_park_lake_state_school,hr.employee_admin,Park Lake State S
...
+employee_resume_chs_davis_plc,hr.employee_chs,Davis PLC,1996-10-05,1997-06-04,resume_type_experience,"Secretary, company"
 employee_resume_qdp_parke_state_school,hr.employee_qdp,Parke State School,1997-06-26,1999-03-17,resume_type_education,
 employee_resume_qdp_evans_cooper_and_white,hr.employee_qdp,"Evans, Cooper and White",1999-04-26,,resume_type_experience,"Therapist, speech and language"
 employee_resume_qdp_rivera_shaw_and_hughes,hr.employee_qdp,"Rivera, Shaw and Hughes",1998-11-26,,resume_type_experience,Landscape architect
-employee_resume_qdp_phillips_jones_and_brown,hr.employee_qdp,"Phillips, Jones and Brown",1999-12-27,1997-07-27,resume_type_experience,"Teacher, special educational needs"
-employee_resume_qdp_hughes_parker_and_barber,hr.employee_qdp,"Hughes, Parker and Barber",1999-02-24,1997-07-27,resume_type_experience,"Engineer, drilling"
+employee_resume_qdp_phillips_jones_and_brown,hr.employee_qdp,"Phillips, Jones and Brown",1999-12-27,2001-07-27,resume_type_experience,"Teacher, special educational needs"
+employee_resume_qdp_hughes_parker_and_barber,hr.employee_qdp,"Hughes, Parker and Barber",1999-02-24,2001-07-27,resume_type_experience,"Engineer, drilling"
```
Comment (Bertrand2) [inline addons/hr_skills/data/hr.resume.line.csv:36]:
```suggestion
employee_resume_qdp_hughes_parker_and_barber,hr.employee_qdp,"Hughes, Parker and Barber",2002-02-24,2007-07-27,resume_type_experience,"Engineer, drilling"

```

File: addons/hr_skills/data/hr.resume.line.csv [hunk 2]
```diff
@@ -3,94 +3,94 @@ employee_resume_admin_park_lake_state_school,hr.employee_admin,Park Lake State S
...
 employee_resume_jve_arnoldcohen,hr.employee_jve,Arnold-Cohen,2003-12-29,,resume_type_experience,Personnel officer
 employee_resume_jep_lawson_public_school,hr.employee_jep,Lawson Public School,1998-06-07,2000-02-17,resume_type_education,
 employee_resume_jep_trinity_college,hr.employee_jep,Trinity College,1995-08-21,1998-04-10,resume_type_education,
 employee_resume_jep_woodend_primary_school,hr.employee_jep,Woodend Primary School,1992-11-22,1995-05-05,resume_type_education,
-employee_resume_jep_mcneil_rodriguez_and_warren,hr.employee_jep,"Mcneil, Rodriguez and Warren",1994-11-22,1993-01-22,resume_type_experience,Sub
-employee_resume_jep_davis_sanchez_and_miller,hr.employee_jep,"Davis, Sanchez and Miller",1995-05-25,1992-11-22,resume_type_experience,Customer service manager
+employee_resume_jep_mcneil_rodriguez_and_warren,hr.employee_jep,"Mcneil, Rodriguez and Warren",1994-11-22,1996-01-22,resume_type_experience,Sub
+employee_resume_jep_davis_sanchez_and_miller,hr.employee_jep,"Davis, Sanchez and Miller",1995-05-25,1997-11-22,resume_type_experience,Customer service manager
```
Comment (Bertrand2) [inline addons/hr_skills/data/hr.resume.line.csv:75]:
```suggestion
employee_resume_jep_mcneil_rodriguez_and_warren,hr.employee_jep,"Mcneil, Rodriguez and Warren",1994-11-22,1996-01-22,resume_type_experience,Sub
employee_resume_jep_davis_sanchez_and_miller,hr.employee_jep,"Davis, Sanchez and Miller",1996-05-25,1997-11-22,resume_type_experience,Customer service manager

```

**Review by Bertrand2 — APPROVED** [review]
A few comments just to make data more coherent while we're at it, otherwise it looks good to me :ok_hand: I'm already greening the task as it's not really an issue

**Comment by sofiagvaladze** [comment]
robodoo r+

**Pull request closed without merging**